RadarSDK/RadarVerificationManager.m

```diff
@@ -288,6 +297,11 @@ - (void)getVerifiedLocationTokenWithCompletionHandler:(RadarTrackVerifiedComplet
     [self trackVerifiedWithBeacons:self.lastTokenBeacons completionHandler:completionHandler];
 }
 
+- (void)setExpectedJurisdictionWithCountryCode:(NSString *)countryCode stateCode:(NSString *)stateCode {
+    self.expectedCountryCode = countryCode;
+    self.expectedStateCode = stateCode;
+}
```

Comment:
We're not persisting the expected jurisdiction, which I kinda expected we would. Could be an issue if `trackVerified` or `startTrackingVerified` is ever called before `setExpectedJurisdiction`.

Also would we want `stateCode` (and maybe even `countryCode` to be a nullable array? I imagine you might want to set an expected `countryCode` without a specific expected `stateCode`, and I also imagine you might want to expect multiple states.

Reply:
The use case here is that some operators do a "geolocation pre-check" on app launch and then select a "site experience" for a single jurisdiction. So (1) it's fine to make them call it on every app launch session and (2) we don't need to support multiple jurisdictions. Happy to share more color on the use case and who needs this offline if you want